db/seeds.rb

```
@@ -138,3 +180,190 @@
+  end
+
+  seeded_teams_with_members = participants_without_teams.each_slice(2).with_index.map do |team_participants, team_index|
+    team = seed_course_report_team(assignment, team_index)
+    seed_course_report_members(team, team_participants)
+
+    [team, team_participants]
+  end
+
+  teams_with_members = existing_teams_with_members + seeded_teams_with_members
+
+  teams_with_members.each_with_index do |(team, _team_participants), team_index|
```

RuboCop (1.86.1)

[warning] Unused block argument - `team_index`. If it's necessary, use `_` or `_team_index` as an argument name to indicate that it won't be used.

(Lint/UnusedBlockArgument)